interpreter/language/parser.go

```diff
@@ -370,12 +370,24 @@ func (p *Parser) parseActions(token Token) []Expression {
 	p.nextToken()
 
 	actions = append(actions, p.parseAction(token))
+	updateTokens := []TokenType{SET, ADD, REMOVE, DELETE}
```

Comment:
Should we validate that an action keyword is only used once? (from the docs, it says an action keyword can only be used once)

https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Expressions.UpdateExpressions.html

Reply:
good catch, I will address it in another PR